_docs/cardano/2017-01-04-transactions.md

```diff
@@ -3,41 +3,113 @@ layout: default
 title: Transactions in Cardano SL
 permalink: /cardano/transactions/
 group: cardano
-visible: false
 ---
 [//]: # (Reviewed at 403cea2d897aba95163b709bd13c35d343116f3f)
 
 # Transactions in Cardano SL
 
-You can think of transactions in Cardano SL as entities that consist of
-a list of inputs and a list of outputs. Outputs of a transaction can later
-be used as inputs for another transaction.
+## Overview
 
-Every node (except for SPV, or lightweight nodes) in the network
-verifies transactions, so those nodes have to keep track of unspent
-outputs, this is called “utxo”, or “Unspent Transaction Outputs”. By
-tracking utxo, every node can validate that inputs in a published
-transaction are indeed the unspent outputs.
+A transaction (_tx_) is a special data represents the _act_ of the value transferring 
```

Comment:
Not quite between two nodes:

* several nodes can have access to the same private key
* money can be transferred to an address that doesn't correspond to any existing private key